ACE/ace/OS_NS_dirent.inl

```diff
@@ -20,8 +20,10 @@ closedir (ACE_DIR *d)
   delete d;
 # elif defined (ACE_HAS_WCLOSEDIR_EQUIVALENT) && defined (ACE_USES_WCHAR)
   ACE_HAS_WCLOSEDIR_EQUIVALENT (d);
-# else /* ACE_WIN32 && ACE_LACKS_CLOSEDIR */
+# elif !defined (ACE_LACKS_CLOSEDIR) /* ACE_WIN32 && ACE_LACKS_CLOSEDIR */
```

Comment:
Find it more readable when you first would check `defined (ACE_LACKS_CLOSEDIR)`, and the normal case in the else part

Reply:
There is a check for `ACE_LACKS_CLOSEDIR` prior to this. I agree with that in general; in this case, however, there isn't such clear separation.